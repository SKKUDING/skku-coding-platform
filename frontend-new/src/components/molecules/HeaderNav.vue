<template>
  <div
    class="
      hidden
      lg:flex
      lg:gap-5
    "
  >
    <NavItem to="/notice">
      Notices
    </NavItem>
    <NavItem to="/contest">
      Contests
    </NavItem>
    <NavItem to="/problem">
      Problems
    </NavItem>
    <NavItem to="/lecture">
      Lectures
    </NavItem>
  </div>
</template>
