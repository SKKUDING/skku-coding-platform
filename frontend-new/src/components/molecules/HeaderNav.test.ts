import { describe, test, expect } from 'vitest'
import { render } from '@testing-library/vue'
import HeaderNav from './HeaderNav.vue'

describe('Molecule - HeaderNav.vue', () => {
  test('should render', () => {
    const { html } = render(HeaderNav)
    expect(html()).toMatchSnapshot()
  })
})
