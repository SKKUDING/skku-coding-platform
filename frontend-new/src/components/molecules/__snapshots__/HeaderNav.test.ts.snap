// Vitest Snapshot v1

exports[`Molecule - HeaderNav.vue > should render 1`] = `
"<div class=\\"hidden lg:flex lg:gap-5\\">
  <router-link class=\\"text-lg font-semibold text-gray-600 hover:text-gray-400 active:text-gray-300 transition hover:ease-in-out\\" active-class=\\"
      text-lime-500
      hover:text-lime-400
      active:text-lime-300
    \\" to=\\"/notice\\">
    <!-- TODO: text color SKKU - #8DC63F --> Notices
  </router-link>
  <router-link class=\\"text-lg font-semibold text-gray-600 hover:text-gray-400 active:text-gray-300 transition hover:ease-in-out\\" active-class=\\"
      text-lime-500
      hover:text-lime-400
      active:text-lime-300
    \\" to=\\"/contest\\">
    <!-- TODO: text color SKKU - #8DC63F --> Contests
  </router-link>
  <router-link class=\\"text-lg font-semibold text-gray-600 hover:text-gray-400 active:text-gray-300 transition hover:ease-in-out\\" active-class=\\"
      text-lime-500
      hover:text-lime-400
      active:text-lime-300
    \\" to=\\"/problem\\">
    <!-- TODO: text color SKKU - #8DC63F --> Problems
  </router-link>
  <router-link class=\\"text-lg font-semibold text-gray-600 hover:text-gray-400 active:text-gray-300 transition hover:ease-in-out\\" active-class=\\"
      text-lime-500
      hover:text-lime-400
      active:text-lime-300
    \\" to=\\"/lecture\\">
    <!-- TODO: text color SKKU - #8DC63F --> Lectures
  </router-link>
</div>"
`;
